spec.bs

```diff
@@ -18,3 +18,122 @@ Default Biblio Status: current
 Markup Shorthands: markdown yes
 Assume Explicit For: yes
 </pre>
+
+
+<h2 id="advertisements">Advertisements</h2>
+
+An {{AuctionAd}} contains the following:
+
+* A <dfn for=AuctionAd dfn-type="dict-member">renderUrl</dfn> .

```

Comment:
I think this should say "render [=URL=]"?

Reply:
When we're defining a field like this, we want the `<dfn>` to include the whole term without embedded markup, and then add ", a [=URL=] that ..." afterward to define the type and meaning. When it's a struct item, the convention is to use space-separated words (and the default [`dfn-type`](https://tabatkins.github.io/bikeshed/#dfn-types) of `dfn`), while when it's a dictionary member, it has to match the exact spelling and capitalization of the IDL.

Reply:
I'm not sure.  The renderUrl name is used as a reference to the name of the property as defined in the idl.  I think there is a notation to lets us link to what we intend (e.g. renderUrl) but display something else (e.g. render).  But I can't find it in the bikeshed documentation.

Is the type of this value any string, as it says (USVString) in the idl, or is the type a string with a specific URL format?